pmm/pmm2-upgrade-tests.groovy

```diff
@@ -265,9 +261,7 @@ pipeline {
             }
             steps {
                 withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'PMM_AWS_DEV', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
-                    setupNodejs()
```

Comment:
is this done already as part of initialization?

Reply:
We already have nodejs in our agent :)